### PR TITLE
Add HDA domain in services seeds

### PR DESCRIPTION
**Pull request by rajivmucheli**

### CODE REVIEW

File: openstack/keppel/templates/seed.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- $domains := list "cc3test" "bs" "cis" "cp" "fsn" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs" }}
+{{- $domains := list "cc3test" "bs" "cis" "cp" "fsn" "hc" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs" }}
```
Comment (Carthaca) [inline openstack/keppel/templates/seed.yaml:1]: should add `hda`

File: openstack/limes/templates/seed.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- $domains := list "bs" "cc3test" "cp" "cis" "fsn" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs" -}}
+{{- $domains := list "bs" "cc3test" "cp" "cis" "fsn" "hc" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs" -}}
```
Comment (Carthaca) [inline openstack/limes/templates/seed.yaml:1]: should add `hda`

**Review by Carthaca — CHANGES_REQUESTED** [review]
Additionally to the inline comments, I would like it better, if the new seeds would be placed alphabetically, i.e. not between 'fsn' and 'hcm', but between 'hcp03' and 'hec'.
But this is just cosmetics in the end

**Review by Carthaca — APPROVED** [review]

**Review by majewsky — APPROVED** [review]

**Review by joker-at-work — APPROVED** [review]

**Review by galkindmitrii — APPROVED** [review]

**Comment by majewsky** [comment]
Limes and Keppel rolled out.

**Comment by Carthaca** [comment]
Manila rolled out

**Pull request merged**